src/components/dashboard/SendReceive/SummaryGeneric.js

```diff
@@ -139,10 +147,8 @@ const SummaryGeneric = ({
           {isBridge && (
             <Section.Row justifyContent="center">
               <View styles={styles.bridgeDesc}>
-                <Section.Text>from your wallet on</Section.Text>
-                <Section.Text>
-                  {network} to your wallet on {altNetwork}
-                </Section.Text>
+                <Section.Text style={{ marginBottom: 10 }}> on {altNetwork} </Section.Text>
+                <Section.Text>{`You'll pay ${formattedFee} G$ in fees to use the bridge`}</Section.Text>
```

Comment:
the fee in % is fee*100/10000
the fee in G$ is amount*fee/10000

Reply:
I think I done as requested